source/sync/data-access-patterns/permissions.txt

```diff
@@ -489,3 +653,26 @@ A user with write permissions for a given collection can modify the value of any
 field of any object matching the client-side query. Write permissions require
 read permissions, so any user that can modify data can also view that data
 before and after it's modified.
+
+.. _flexible-sync-field-level-permissions:
+
+Field-Level Permissions
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can refine document-level permissions on a per-field basis with the
+``fields`` and ``additional_fields`` properties of the **Role** configuration
+object.
+
+If the user does not have document-level permissions to a document, field-level
+permissions cannot grant access to that document. Field-level permissions can
+only refine the permissions on documents that a user can access. For example,
+you can use field-level permissions to forbid writes on a specific field of a
+document that a user can otherwise write to.
+
+``additional_fields`` defines the field-level permissions for fields that you
+don't explicitly define in the ``fields`` object. If you don't define
+``additional_fields`` read or write, the system uses the corresponding
+document-level permission for any field not mentioned in ``fields``.
+
+In practice, you would use ``additional_fields`` with ``fields`` to greatly
```

Comment:
This reads a bit like you're saying "If you know what you're doing, you'll use both properties". Also, "greatly constrain" is perhaps better as "constrain to a higher degree" or "constrain to a finer level of detail"